codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketProtocolHandler.java

```diff
@@ -110,13 +111,12 @@ public void write(final ChannelHandlerContext ctx, Object msg, ChannelPromise pr
         if (closeSent != null) {
             ReferenceCountUtil.release(msg);
             promise.setFailure(new ClosedChannelException());
-            return;
```

Comment:
@normanmaurer do we risk an NPE here? https://github.com/netty/netty/pull/10248/files#diff-1eb10f9889aa482b57792f4cb001aa2dL99

Reply:
![Screenshot from 2020-05-05 17-41-36](https://user-images.githubusercontent.com/323497/81085637-c320e280-8ef7-11ea-8e4a-69b907746182.png)

Reply:
nope as `write` will set `closeSent` I think ?